game-core/src/test/java/games/strategy/triplea/ui/logic/RouteCalculatorTest.java

```diff
@@ -156,7 +156,8 @@ public void testGetAllNormalizedLines() {
     final double[] testData = new double[1000];
     Arrays.setAll(testData, Double::valueOf);
     final List<Path2D> paths = routeCalculator.getAllNormalizedLines(testData, testData);
-    final Iterator<AffineTransform> transforms = routeCalculator.getPossibleTranslations().iterator();
+    final Iterator<AffineTransform> transforms = MapScrollUtil.getPossibleTranslations(
+        true, true, 1000, 1000).iterator();
```

Comment:
To avoid accidental drift between these arguments and those used to create the `RouteCalculator` above, would it be better to extract local constants in this method to avoid the duplication?  Or alternatively add getters to `RouteCalculator` and call those getters on this line instead of using the constants/literals?

Reply:
Because I'm using 1000 as constant expression all over this class, I'll just put the 1000s into a constant.
The true false things are different in every test, so they shouldn't really matter that much IMO

Reply:
Actually, my bad for not even realizing I was in test code.  NBD then if you want to skip it because the test will probably fail if one of the literals is changed in one place and not the other.